### DISCUSSION TITLE
Teradak ES9023 v2.1 DAC Module

### DISCUSSION THREAD

**Discussion by luisfcorreia** [probonopd/MiniDexed#191] [Show and tell]
Hi everyone,

yesterday I managed to successfully test this module I remember having around here.

Although this uses a different chip, it's fully compatible.

[Teradak ES9023 v2.1](http://www.teradak.com/products/84.html)

Pictured here connected to my Raspberry Pi 1B, with LCD, encoder and Classic MIDI input circuit on breadboard

![IMG_20220428_223436-01](https://user-images.githubusercontent.com/191885/165913989-93181100-d196-4617-9a49-9c5b325c78a0.jpeg)

**Comment by probonopd** [comment]
Thanks for letting us know, added to the wiki.
